src/View/Helper/MigrationHelper.php

```diff
@@ -339,8 +339,14 @@ public function primaryKeysColumnsList($table)
      */
     public function column($tableSchema, $column)
     {
+        $columnType = $tableSchema->getColumnType($column);
+        // Phinx doesn't understand timestampfractional.
+        if ($columnType === 'timestampfractional') {
+            $columnType = 'timestamp';
```

Comment:
Must be some way to specify the proper type in phinx.

Reply:
I don't think phinx has support for the other postgres specific timestamp types.

Reply:
Interesting. That type isn't restricted to postgres. Does phinx not let you set the precision length?